dusseldorf-ktor-auth/src/main/kotlin/no/nav/helse/dusseldorf/ktor/auth/TokenUtils.kt

```diff
@@ -46,3 +55,4 @@ fun ApplicationCall.idToken() : IdToken {
 
 class CookieNotSetException(cookieName : String) : RuntimeException("Ingen cookie med navnet '$cookieName' satt.")
 class IdTokenInvalidFormatException(idToken: IdToken, cause: Throwable? = null) : RuntimeException("$idToken er på ugyldig format.", cause)
+class NoTokenSetException() : RuntimeException("Fant ikke token som cookie eller authorization header.")
```

Comment:
Denne kan du kalle for UnAuthorizedException. 
Husk at denne exception her også må håndteres i IdTokenStatusPages.

Reply:
Okei. Ja fant ut det etterhvert..